src/main/config/messages_template.xml

```diff
@@ -423,6 +423,11 @@ See the accompanying LICENSE file for applicable license.
     <response></response>
   </message>
 
+  <message id="DOTJ084E" type="ERROR">
+    <reason>Included file %1 contained invalid byte sequence for charset %2</reason>
+    <response></response>
```

Comment:
@infotexture Could you help with the message text? Is this understandable to our users? They probably don't understand what "invalid byte sequence means".  Should we go for less programmery

```xml
<reason>Failed to read file %1 with charset %2.</reason>
<response>Check that file is saved as %2.</response>
```

Which would result in something like

> Failed to read file file://…/topic.dita with charset UTF-8. Check that file is saved as UTF-8.